test/fixture/locales/US/en/complex.properties

```diff
@@ -0,0 +1,5 @@
+value=A Value with {numPhotos, plural, =0 {no photos.} =1 {one photo.} other {# photos.}}
+array[0]=First Entry
+array[1]=Second Entry
+map[US]=United States
+map[GB]=United Kingdom
```

Comment:
Thanks for taking a look @mimckenna. I don't want to duplicate the unit testing done in the `yahoo-intlmessageformat` module itself. That said, I'm sure there are other aspects that should be tested with respect to this integration.